function-server/src/main/java/io/dispatchframework/javabaseimage/Dummy.java

```diff
@@ -1,10 +0,0 @@
-package io.dispatchframework.javabaseimage;
```

Comment:
Did you mean to delete this class?

Reply:
There's just a lot of unused code that wasn't removed in the source dir update. That Dummy class doesn't seem to be used.

Reply:
Ok, that's fine with me.